### DISCUSSION TITLE
Add description for the RESET symbol.

### DISCUSSION THREAD

**Discussion by MiroslavPetrik** [pmndrs/jotai#2306] [Ideas] [answered]
Currently when the `RESET` symbol is logged in console, it displays only `Symbol()`.
However, the [`Symbol()` accepts a description](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol/Symbol#description).

I suggest to add simple description, so the debugging is easier:

`export const RESET = Symbol("RESET")`

**Comment by himself65** [comment]
good idea, you can have a PR for this!

**Comment by dai-shi** [comment]
Not having symbol description was intentional.
Out of curiosity, how did you come across to console log the symbol? What was your use case?

**Reply by MiroslavPetrik** [reply]
I have a custom resettable atom & I was debugging it's setter.
https://github.com/form-atoms/field/blob/3c4a5cca7409475521a90154e3c5d7021740603e/src/atoms/list-atom/listAtom.ts#L118

**Reply by dai-shi** [reply]
Okay, so you are a library author. I understand the use case.

**Comment by himself65** [comment, marked as answer]
https://github.com/pmndrs/jotai/pull/2307
